reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt

```diff
@@ -29,11 +29,19 @@ import io.kotlintest.specs.WordSpec
 
 import java.io.File
 
+import javax.xml.transform.TransformerFactory
+
 class StaticHtmlReporterTest : WordSpec({
     val ortResult = File("../scanner/src/funTest/assets/file-counter-expected-output-for-analyzer-result.yml")
             .readValue<OrtResult>()
 
     "StaticHtmlReporter" should {
+        "use the Apache Xalan 2.7.2 TransformerFactory" {
```

Comment:
We do not really test for the version here (only that it's not the version bundled with the JDK), so should it be part of the test name?

Reply:
Removed.